### DISCUSSION TITLE
Why is AdMob package removed from v11.5.0?

### DISCUSSION THREAD

**Discussion by Satya5614** [invertase/react-native-firebase#5326] [Q&A]
I am not sure why the AdMob package is removed from the latest version(v11.5.0) of react-native-firebase.

**Comment by mikehardy** [comment]
AdMob is not removed from v11.5.0. v11.5.0 contains AdMob.

It will be removed from v12 though - it has been removed from Firebase - you may look upstream for confirmation and news on that in the firebase-android-sdk and firebase-ios-sdk release notes, which you should in general stay familiar with.

We intend to re-host the code in an `@invertase/react-native-admob` package with APIs intact but as with our plans for the new google MLKit we have quite a bit of firebase-core work to do before we can handle related repos, please have patience

**Reply by mikehardy** [reply]
This came up as a comment on an unrelated issue and I think I may understand the confusion now - pasting in my comment from there:

----

Ah - sorry, we've already merged the change where Admob is removed from firebase following it's removal upstream, and the documentation site goes live immediately after merges to main branch.

It is not as great of an experience but for now until we have the admob code re-hosted in a non-firebase repo you can go direct to the usage docs and types / API reference from the v11.5.0 (current stable) tag

https://github.com/invertase/react-native-firebase/tree/v11.5.0/docs/admob
https://github.com/invertase/react-native-firebase/blob/v11.5.0/packages/admob/lib/index.d.ts

Normally it's a great thing that the docs site is updated immediately between releases, but in this case it means the documentation for a module that is still live (and will always be, in that version) is gone. I'll have to think on that to see if there's some better way we can keep docs for old versions alive as they are still quite useful
